packages/syft/src/syft/service/action/action_object.py

```diff
@@ -1195,11 +1205,13 @@ def wrapper(
 
         return wrapper
 
-    def send(self, client: SyftClient) -> Self:
+    def send(self, client: SyftClient, add_storage_permission: bool = True) -> Self:
```

Comment:
I get that this is here, but at the same time I would have hoped that we could hide this complexity from such an essential user facing api

Reply:
maybe we can add a ._send() which has this argument, but .send() does not?